### ISSUE TITLE
Discrepancy with google group names

### ISSUE THREAD

**Issue by malet-datatonic** [GoogleCloudPlatform/cloud-foundation-fabric#852]
Within the "Set up your foundation" section, we don't have the `gcp-support` group by default, and we have the additional gcp-developers group:
![image](https://user-images.githubusercontent.com/89927664/193585328-eee4db2e-e1b6-495a-add8-75c66d0a40d4.png)

I assume the solution is to update the "Set up your foundation" wizard to match the groups defined in this repository (to have a unified approach to bootstrap).

**Comment by juliocc** [comment]
Indeed `gcp-support` is not part of the checklist. However, we list it as a requisite in the [bootstrap documentation](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/fast/stages/00-bootstrap#groups).

**Comment by Malet** [comment]
The documentation points to the "Set up your foundation" wizard as a reference:
> You can refer to [this animated image](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/fast/stages/00-bootstrap/groups.gif) for a step by step on group creation. 

As a newcomer to this repository, I had made the assumption that the listed groups in the CFF docs matched the list in the wizard.

I think since `gcp-support` is a special case, above the ones created by default this should be highlighted as an additional requirement.

**Comment by juliocc** [comment]
Good point. I've updated the documentation to make it explicit that gcp-support needs to be created manually.
